### PR TITLE
Next Chunk Doesnt Contain Data

### PR DESCRIPTION
**Pull request by MoriaYechiel**
The first call to getChunk does not retrieve any data

### CODE REVIEW

**Comment by bryaakov** [comment]
what if user was providing `&with-data`? will it work, or does they must specify `&with-data=true`?

**Comment by MoriaYechiel** [comment]
@bryaakov &with-data is absolutely enough

File: server/cmwell-ws/app/logic/CRUDServiceFS.scala [hunk 1]
```diff
@@ -817,10 +818,21 @@ class CRUDServiceFS @Inject()(implicit ec: ExecutionContext, sys: ActorSystem) e
...
+      case false =>
+        searchResultFuture.map { ftsResults =>
+          val response = ftsResults.response
+          IterationResults(response.scrollId, response.total, Some(response.infotons), debugInfo = ftsResults.searchQueryStr)
+        }
+      case true =>
+        searchResultFuture.flatMap { ftsResults =>
+          val infotons = enrichInfotonsData(ftsResults.response.infotons)
+          val response = ftsResults.response
+          infotons.map{infoton => IterationResults(response.scrollId, response.total, Some(infoton), debugInfo = ftsResults.searchQueryStr)}
```
Comment (e-orz) [inline server/cmwell-ws/app/logic/CRUDServiceFS.scala]: change to `infotonSeq` or something

File: server/cmwell-ws/app/logic/CRUDServiceFS.scala [hunk 2]
```diff
@@ -877,31 +889,36 @@ class CRUDServiceFS @Inject()(implicit ec: ExecutionContext, sys: ActorSystem) e
...
-                logger.error(
-                  "some uuids retrieved from ES, could not be retrieved from cassandra: " + esUuidsSet
-                    .diff(casUuidsSet)
-                    .mkString("[", ",", "]")
-                )
-          }
-          val infotons = addIndexTime(infotonsSeq.collect{case FullBox(i) => i}, ftsResults.infotons)
-          IterationResults(ftsResults.scrollId, ftsResults.total, Some(infotons), debugInfo = ftsResults.searchQueryStr)
+          val infotons = enrichInfotonsData(ftsResults.infotons)
+          infotons.map{infoton => IterationResults(ftsResults.scrollId, ftsResults.total, Some(infoton), debugInfo = ftsResults.searchQueryStr)}
```
Comment (e-orz) [inline server/cmwell-ws/app/logic/CRUDServiceFS.scala]: same

**Pull request merged**